src/assignment_io.py

```diff
@@ -298,13 +298,14 @@ def add_read_info(self, read_assignment):
 class SqantiTSVPrinter(AbstractAssignmentPrinter):
     def __init__(self, output_file_name, params, io_support):
         AbstractAssignmentPrinter.__init__(self, output_file_name, params)
-        self.header = 'isoform\tchrom\tstrand\tlength\texons\tstructural_category' \
+        self.header = '#isoform\tchrom\tstrand\tlength\texons\tstructural_category' \
```

Comment:
SQANTI does not have # in header https://raw.githubusercontent.com/ConesaLab/SQANTI3/master/example/SQANTI3_QC_output/UHR_chr22_classification.txt Also, the header has slightly different columns